src/App/Support/Enpii_Base_Bootstrap.php

```diff
@@ -7,38 +7,27 @@
 use Enpii_Base\App\Support\Enpii_Base_Helper;
 
 class Enpii_Base_Bootstrap {
-	public static function initialize() {
-		// Check if the WordPress content directory is loaded.
+	public static function initialize( string $plugin_url, string $dirname ) {
 		if ( Enpii_Base_Helper::is_wp_content_loaded() ) {
-	
-			// If the current mode is the command-line interface (CLI) mode.
-			if ( Enpii_Base_Helper::is_console_mode() ) {
-				// Register the CLI initialization action.
-				static::register_cli_init_action();
-	
-				// If the command 'enpii-base prepare' is present, prepare the WP App folders.
-				if ( static::is_enpii_base_prepare_command() ) {
-					Enpii_Base_Helper::prepare_wp_app_folders();
-	
-					// Return early after preparing the folders since no further actions are needed.
-					return;
-				}
-			} 
-	
-			// Check if the wp_app setup has been done correctly
-			// If the check fails, return early and do not proceed with further setup.
-			if ( ! Enpii_Base_Helper::perform_wp_app_check() ) {
+			static::register_cli_init_action();
+		
+			if ( ! Enpii_Base_Helper::is_console_mode() && ! Enpii_Base_Helper::perform_wp_app_check() ) {
+				// We do nothing but still keep the plugin enabled
 				return;
 			}
-	
-			// Register a redirect to the application setup page.
-			static::register_setup_app_redirect();
-	
-			// Register the WP App setup hooks.
-			static::register_wp_app_setup_hooks();
-	
-			// Register the action to signal that the WP App has fully loaded.
-			static::register_wp_app_loaded_action();
+		
+			if ( ! Enpii_Base_Helper::is_console_mode() ) {
```

Comment:
if we decide to use the Enpii_Base_Helper, we should move Enpii_Base_Bootstrap to this class to avoid the confusion usages both classes